<?xml version="1.0"?>
<launch>
<!--This launchfile should contain important nodes that we should always have
running, such as the UPS power monitor. This launchfile will be run as a system
service during computer startup via the ROS robot_upstart package -->

    <include file="$(find robosub)/launch/movement.launch"/> 
    <include file="$(find robosub)/launch/sensors.launch"/> 
    <include file="$(find robosub)/launch/cameras.launch"/> 
    <node pkg="robosub" name="undistortion" type="undistortion" output="screen"/>
   <!-- <include file="$(find robosub)/launch/vision.launch"/> --> 
   <!--  <node pkg="robosub" name="keyboard_control" type="keyboard_control" output="screen"/> -->
</launch>
